app/api/file/[name]/[tool]/route.ts

```diff
@@ -1,4 +1,6 @@
+
 export const dynamic = 'force-dynamic' // defaults to autover'
+import {NextResponse} from "next/server";
```

Comment:
My linter was complaining about `Response.json` not existing here.

Reply:
Fair, it is auto imported I believe. Also this whole API section can get removed soonish. It isn't used right now, not important but just adjacent.